development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
+* Open the top level ``CMakeLists.txt`` in QtCreator
+
+### Making a kit
+
+You need to modify the desktop kit to use your brew install qt5:
+
+``Preferences -> build and run -> Qt Versions -> Manual -> Add``
+
+Set the executable to :
+
+``/usr/local/bin/qmake``
+
+``Preferences -> build and run -> Cmake -> Manual -> Add``
+
+* **Name:** Brew CMake
+* **Path:** /usr/local/bin/cmake
+
+
+``Preferences -> build and run -> Kits -> Manual Desktop``
+
+Press clone to make a copy and in the copy set it up with the following options:
+
+* **Name:** QGIS Build Kit - Qt5
+* **File system name:** Leave blank
+* **Device type:** desktop
+* **Device:** Local PC
+* **Sys root:** Leave blank
+* **Compiler:** Clang (x86_64bin in /usr/bin)
+* **Environment:** click ‘Change …’  and add this to your python path so sip can be found:
+
+``export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.5/site-packages``
```

Comment:
I just experimented with prepending the env var configuration with `export` and proofing the env during build. This results in a _literal_ `export var=value` line in `env` output. So, I think `export` is not supported in [Qt Creator's "Batch editing" text field](http://doc.qt.io/qtcreator/creator-build-settings.html#batch-editing). For example, adding it to PATH results in a duplicate line:

```
PATH=/usr/local/...
export PATH=/usr/local/...
```

Unless I am incorrect, such an `export` _prefacing_ an internal env line will not actually export the variable, i.e. it is treated as text. Or will it? (My BASH knowledge is thin here.)

Also, according to Creator's manual for Batch env var editing, variables ostensibly _should_ be able to be referenced in later config lines, but I have not found this to be the case on macOS.